lsp-ltex.el

```diff
@@ -50,7 +50,8 @@ https://github.com/valentjn/ltex-ls"
 
 (defcustom lsp-ltex-active-modes
   '( text-mode
-     bibtex-mode context-mode latex-mode
+     bibtex-mode context-mode
+     latex-mode LaTeX mode ;; AUCTeX 14+ has renamed latex-mode to LaTeX-mode
```

Comment:
You are missing a - here.

Reply:
Thanks for spotting this, I did the fix quickly on GitHub and didn't even consider it as a possible mistake